### PR TITLE
Add Out Of Office support

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
Add constants for mobile and webapp support
* OUT_OF_OFFICE = ooo
* SYSTEM_AUTO_RESPONDER = system_auto_response

via @miguelespinoza

### CODE REVIEW

File: src/constants/general.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default {
 
     MENTION: 'mention',
 
+    OUT_OF_OFFICE: 'ooo',
```
Comment (jespino) [inline src/constants/general.js]: I don't love the `ooo` value, why not `out_of_office`?

Comment (cpanato) [reply src/constants/general.js]: +1 for ooo 😄

Comment (miguelespinoza) [reply src/constants/general.js]: @jespino I followed the convention with DND.  `dnd`.  From my assumption, it seemed like the reasoning for `do_not_disturb` felt inadequate?

Therefore I went with `ooo`.

Also, from a feature standpoint, slash commands follow the same format /dnd, therefore once slash command for Out Of Office is implemented we would use `/ooo`

Comment (jespino) [reply src/constants/general.js]: ok, makes sense :)

**Comment by miguelespinoza** [comment]
@jespino if this looks good can we merge so clients can make use of the changes.

Thanks! 😄

**Comment by jespino** [comment]
I'm ok with that, but I'm waiting for @hmhealey approval :)

**Pull request merged**